data_inconsistencies.py

```diff
@@ -1,5 +1,6 @@
 import os
 import csv
+from autowebcompat import utils
 
 
 def get_inconsistencies():
```

Comment:
This function should use utils.get_all_images() too, but it should be changed in a separate PR.
Can you do it?

Reply:
@marco-c  wouldn't it just be writing ``` files = utils.get_all_images() ``` instead of ```listdir(data)``` ? or is there a corner case to think about ?

Reply:
Yes, just that

Reply:
It should be done in a separate PR.